### ISSUE TITLE
Java 6.1: If-else blocks

### ISSUE THREAD

**Issue by AkMo3** [girlscript/winter-of-contributing#1807]
### Description

Add audio, documentation, and video regarding the **If - else blocks**.

____

### Location of File to be added

The files should be added inside the `If - else blocks`  folder which is inside the `Control Statements`  folder present in the   `Java`  main folder.  The documentation file can be in the `.md`/`.py`/`.ipynb` format. Audio should be in `.mp3` format and to be submitted through a Google drive link. Video can be in any format and should be submitted through a Google Drive link.

 **Note:**
 - You are required to comment with `\assign CONTENT_TYPE`, so that other content types can be created at the same time.
 - Failing to comment in the above manner will lead to the removal of assignment from the issue even if the bot assigns.
 - You are required to do a PR within 7 days otherwise you will be unassigned and others will be assigned.
 - Issues will be assigned on a first come first serve basis.

### Domain

Java_Domain

### Type of Contribution

Audio, Video, Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by RahulJaiswal-18** [comment]
\assign Documentation

**Comment by astharaj300** [comment]
\assign Documentation

**Comment by shreya40** [comment]
\assign document

**Comment by pragatikesarwani38** [comment]
I would like to contribute to this via documentation @AkMo3

**Comment by AkMo3** [comment]
@pragatikesarwani38 Documentation content has already been assigned. Only audio and video is available.

**Comment by AkMo3** [comment]
@RahulJaiswal-18 You have been assigned documentation for this issue.

**Comment by Khushi-Agrawal-1** [comment]
\assign Audio
I would like to contribute to this via audio @AkMo3

**Comment by garvitraj** [comment]
@Khushi-Agrawal-1 go with the audio

**Comment by Shraddha-Jaggi** [comment]
\assign video

**Comment by satyampadhi2000** [comment]
\assign Document

**Comment by Shraddha-Jaggi** [comment]
\assign video

**Comment by Khushi-Agrawal-1** [comment]
@AkMo3 I'm little confused as to what to do with my Google Drive link. I'm contributing with audio but I have also made a markdown file just for reference purpose so am I suppose to put the link in it

**Comment by Khushi-Agrawal-1** [comment]
@garvitraj I'm little confused as to what to do with my Google Drive link. I'm contributing with audio but I have also made a markdown file just for reference purpose so am I suppose to put the link in it ?

**Comment by AkMo3** [comment]
> 
> 
> @garvitraj I'm little confused as to what to do with my Google Drive link. I'm contributing with audio but I have also made a markdown file just for reference purpose so am I suppose to put the link in it ?

Yes. Please see https://github.com/girlscript/winter-of-contributing/pull/3157/files for reference

**Comment by Khushi-Agrawal-1** [comment]
Hey @AkMo3 can you please merge the PR as I'm not able to generate PR for another issue under Java. Thankyou
